apps/penxle.com/src/routes/(default)/[space]/dashboard/settings/+page.svelte

```diff
@@ -180,50 +190,44 @@
           class="border border-secondary square-6 bg-white text-secondary rounded-lg flex center"
           type="button"
           on:click={() => {
-            links.push({ id: links.length + 1, url: '' });
+            links.push({ idx: links.length, link: '' });
             links = links;
           }}
         >
           <i class="i-lc-plus" />
         </button>
       </p>
 
-      <ul class="space-y-3">
-        <!-- {#each links as link, idx (link)}
-          <li
-            class="flex items-start gap-3"
-            draggable="true"
-            on:dragstart={({ dataTransfer }) => onDragStart(dataTransfer, link)}
-            on:dragover|preventDefault={() => null}
-            on:drop|preventDefault={() => onDrop(link)}
-            animate:flip={{ duration: 500 }}
-          >
+      <ul class="externalLinks space-y-3">
+        {#each links as { link, idx }, index (idx)}
+          <li class="flex items-start gap-3" data-id={link}>
             <div>
               <button
                 class="square-6 p-0.5 border border-secondary rounded-lg flex center mb-1 transition cursor-move hover:border-primary"
                 type="button"
-                on:dragstart={({ dataTransfer }) => onDragStart(dataTransfer, link)}
-                on:dragover|preventDefault={() => null}
               >
-                <i class="i-lc-menu square-3.5 text-icon-secondary" />
+                <i class="linkHandle i-lc-menu square-3.5 text-icon-secondary" />
               </button>
               <button
                 class="square-6 p-0.5 border border-secondary rounded-lg flex center transition hover:border-primary"
                 type="button"
                 on:click={() => {
-                  links = links.filter((l) => l.id !== link.id);
+                  links = links.filter((_, i) => i !== index);
                   links = links;
                 }}
               >
                 <i class="i-lc-trash square-3.5 text-red-50" />
               </button>
             </div>
 
-            <FormField name={`url${idx + 1}`} class="grow" label={`URL ${idx + 1}`}>
-              <TextInput class="w-full" placeholder="URL을 입력해주세요" bind:value={link.url} />
-            </FormField>
+            <div
+              class="w-full flex flex-col gap-1.5 border border-gray-5 rounded-2xl transition bg-primary pt-3 pb-4 px-3.5 hover:border-secondary focus-within:border-tertiary! [&:has(textarea[aria-invalid])]:border-action-error disabled:opacity-50"
+            >
+              <label class="body-13-b" for={`url${index + 1}`}>{`URL ${index + 1}`}</label>
+              <TextInput class="w-full" placeholder="URL을 입력해주세요" bind:value={link} />
```

Comment:
P3: <TextInput /> 에 `id` 와 `type="url"` 어트리뷰트 추가를 웬만하면 반영해주세요!

```suggestion
              <label class="body-13-b" for={`url${index + 1}`}>{`URL ${index + 1}`}</label>
              <TextInput id={`url${index + 1}`} type="url" class="w-full" placeholder="URL을 입력해주세요" bind:value={link} />
```

Reply:
올렸다고 생각했는데 안올라가있었네요..! 수정했습니다 :)

Reply:
P4: `autocomplete` 어트리뷰트도 달아주시면 이전에 사용자가 브라우저에 입력한 URL 정보를 재사용할 수 있어 편하지 않을까 싶네요..!